Util/PlantDetector.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+from pprint import pprint
+
+from Exceptions.PlantsUndetectedError import PlantsUndetectedError
+
+
+def detect(image_path: str):
+    with open("plantnet_API_key", "r") as file:
+        API_KEY = file.readline()
+
+    PROJECT = "all"  # try "weurope" or "canada"
+    api_endpoint = f"https://my-api.plantnet.org/v2/identify/{PROJECT}?api-key={API_KEY}"
+
+    data = {'organs': ['flower']}
```

Comment:
> Organs associated to images (must contains one of: leaf, flower, fruit, bark, auto. Could contains: habit, other)

I'm not totally sure what this does.
Does this mean we only ID the first plant with a flower?

Reply:
I think it means that the API is looking for flowers in the image. 

When I update the processing to store results that pass a threshold I can also update this to also check for leafs, fruit, and bark

The API sends back a list of results saying what it thinks the image contains, I think it's designed for 1 plant but I can't see why it doesn't work with multiple, I'll also do some testing for that

Reply:
Worth a TODO note then?

Reply:
Added